tests/cbmc/include/cbmc_proof/make_common_datastructures.h

```diff
@@ -19,13 +19,33 @@
 #include "api/s2n.h"
 #include "stuffer/s2n_stuffer.h"
 
+/*
+ * Checks whether s2n_blob is bounded by max_size.
+ */
+bool s2n_blob_is_bounded(const struct s2n_blob* blob, const size_t max_size);
+
+/*
+ * Ensures s2n_blob has a proper allocated data member.
+ */
 void ensure_s2n_blob_has_allocated_fields(struct s2n_blob* blob);
+
+/*
+ * Properly allocates s2n_blob for CBMC proofs.
+ */
 struct s2n_blob* cbmc_allocate_s2n_blob();
+
+/*
+ * Ensures s2n_stuffer has a proper allocated blob member.
+ */
 void ensure_s2n_stuffer_has_allocated_fields(struct s2n_stuffer* stuffer);
+
+/*
+ * Properly allocates s2n_stuffer for CBMC proofs.
+ */
 struct s2n_stuffer* cbmc_allocate_s2n_stuffer();
 
-/**
```

Comment:
Usually I see `*` in the codebase, but I don't believe there is a pattern across the entire project.

Reply:
Yeah in terms of formatting, I've been less picky in anticipation of `clang-format` changes getting merged and required. I'm assuming something like this will be fixed by it, but I would have to look.